mypyc/ops_misc.py

```diff
@@ -29,6 +29,23 @@ def emit_none(emitter: EmitterInterface, args: List[str], dest: str) -> None:
                        error_kind=ERR_NEVER,
                        emit=simple_emit('{dest} = 0;'))
 
+iter_op = func_op(name='builtins.iter',
+                  arg_types=[object_rprimitive],
+                  result_type=object_rprimitive,
+                  error_kind=ERR_MAGIC,
+                  emit=simple_emit('{dest} = PyObject_GetIter({args[0]});'))
+
+next_op = func_op(name='builtins.next',
+                  arg_types=[object_rprimitive],
+                  result_type=object_rprimitive,
+                  error_kind=ERR_NEVER,
```

Comment:
Add a comment that despite it being `ERR_NEVER`, it really can return NULL and needs to be checked with `Branch.IS_ERROR`.